zoom-meeting-sdk-linux/h/meeting_service_components/meeting_recording_interface.h

```
@@ -1,25 +1,31 @@
-/*!
-* \file meeting_recording_interface.h
-* \brief Recording of Meeting Service Interface
-* Valid for both ZOOM style and user custom interface mode.
-*/
+/**
+ * @file meeting_recording_interface.h
+ * @brief Recording of Meeting Service Interface
+ * @note Valid for both ZOOM style and user custom interface mode.
+ */
 #ifndef _MEETING_Recording_INTERFACE_H_
 #define _MEETING_Recording_INTERFACE_H_
```

Clang (14.0.6)

[warning] declaration uses identifier '_MEETING_Recording_INTERFACE_H_', which is a reserved identifier

(bugprone-reserved-identifier)